src/fty_mc_server.cc

```diff
@@ -489,10 +489,10 @@ fty_mc_server_test (bool verbose)
 {
     printf (" * fty_mc_server:");
     printf ("\n");
-    assert(fty_shm_set_test_dir("src/selftest-rw") == 0);
+    assert(fty_shm_set_test_dir("selftest-rw") == 0);
```

Comment:
prefer to use a #define or a const char*  for 'selftest_rw'

Reply:
Take into account